LEAF_Request_Portal/mailer/mailer.php

```diff
@@ -62,15 +62,20 @@
             if (strlen(trim($email['recipient'])) == 0)
             {
                 // delete invalid cache
-                unlink($folder . $item);
+                if (file_exists($folder . $item)) {
+                    unlink($folder . $item);
+                }
+
                 trigger_error('Mail no recipient: ' . $email['subject']);
             }
             else
             {
                 touch($folder . $item);    // reset timer
                 if (mail($email['recipient'], $email['subject'], $email['body'], $email['headers']))
                 {
-                    unlink($folder . $item);
+                    if (file_exists($folder . $item)) {
+                        unlink($folder . $item);
+                    }
```

Comment:
I was seeing a couple different things going on here. The file_esixts might need to go on the outside of the if(mail and touch 70/73 above. Similar to line 37, 

I was trying to figure out why the touch was failing to find the file which might be a condition where two people are triggering the email sending close to the same time. The error was happening a lot in the morning which could lend to people running through reminding folks or moving something through the workflow. This second part is the thing that I am uncertain of.

Reply:
This is more for Pete maybe, if we move it outside the if statement then the trigger error won't get hit. I know Pete you said this could use some looking over, should we back this out and take a deeper dive into it or do we want to get this out to remove those errors? I can move the if if that seems better.

Reply:
Yeah the error wont get triggered but that just means its already sent or gone. So there shouldn't be an error.